src/csync/csync_exclude.cpp

```diff
@@ -330,8 +323,17 @@ bool ExcludedFiles::reloadExcludeFiles()
     bool success = true;
     const auto keys = _excludeFiles.keys();
     for (const auto& basePath : keys) {
-        for (const auto& file : _excludeFiles.value(basePath)) {
-            success = loadExcludeFile(basePath, file);
+        for (const auto& excludeFile : _excludeFiles.value(basePath)) {
```

Comment:
I know it was here before, but since we're touching the line anyway, I'd fix the placement of the space around & (so space before and not after)